packages/contracts/src/networks.ts

```diff
@@ -432,13 +432,6 @@ export const SPHINX_NETWORKS: Array<SupportedNetwork> = [
         browserURL: 'https://lineascan.build',
         envKey: 'LINEA_ETHERSCAN_API_KEY',
       },
-      blockscout: {
-        apiURL: 'https://explorer.goerli.linea.build/api',
-        browserURL: 'https://explorer.goerli.linea.build/',
```

Comment:
Linea no longer has a Blockscout instance for mainnet. Also, this configuration was wrong anyway.